### ISSUE TITLE
Bismuth high in urine

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10394]
**Preferred term label:**
Bismuth high in urine

**Synonyms**

**Definition (free text, please give PubMed ID)**
greater than 20µg/dL

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Bismuth toxicity

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000888
- label: Elevated urine bismuth level
- parents: [Urine xenobiotic (HP:0031840)](https://hpo.jax.org/app/browse/term/HP:0031840)
- pmid: 
- def: Detection of a level of bismuth in the urine high enough to raise the suspicion of bismuth toxicity, usually greater than 20microgram per deciliter.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10394
- synonyms:
